public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,103 @@
+# dvc.api.get_url()
+
+Returns the URL to the storage location of a data file or directory tracked by
+DVC.
+
+## Definition
+
+### Signature
```

Comment:
this split into Signature, Types looks cumbersome ... we need to come with some simple convention (kind-a docstring, or Python type annotations) and make a single section similar to CR

Reply:
This was changed per Alex' suggestion to put basic usage on top:

![image](https://user-images.githubusercontent.com/1477535/75293063-028dcd00-57eb-11ea-9c81-057e72647375.png)

^ Still cumbersome? I could remove the **Signature** header and move the actual signature to the top, just after the short definition under title.

Reply:
ideally one short section, may be utilize Python types notation

Reply:
I would check other docs (not only stdlib)

Reply:
I think most Python libs refs have a page by module ([1](https://docs.python.org/3.7/library/types.html#standard-interpreter-types), [2](https://docs.paramiko.org/en/stable/api/sftp.html)), and they just list each function with some description inside. Only more complex ones have examples, etc. (Classes have longer sections each, with subsections for each prop and method.) We have a page per function and lots more info than that, but what I came up with is this:

![image](https://user-images.githubusercontent.com/1477535/75422334-f0528280-5901-11ea-8830-fb635d07ecb1.png)

Which may need some CSS styling to make it look better. The title is now an H3 FYI. Also will probably further edit how types and exceptions are explained.

Reply:
Still have very mixed feeling about this layout, types, signature, return value, exceptions ... these comments in the usage example. We need to do something with it. Let's discuss in Slack.